### ISSUE TITLE
Error when exporting aab file after changing release.keystore to release.jks

### ISSUE THREAD

**Issue by GrAfDeveloper** [godotengine/godot#69337]
### Godot version

3.5.1

### System information

Windows 10

### Issue description

I have an app uploaded to the Google Play Console and I wanted to update it. But due to carelessness I lost the old release.keystore and I decided to replace it. To replace it, I used the new .jks format because Google Play Console asked for a .pem file (in order to create a .pem file, you must have a .jks file).
And now when exporting the aab file in godot, this problem occurs. Everything is normal when exporting apk. What will be the ideas to solve the problem?

### Steps to reproduce

Starting a Gradle Daemon, 3 busy and 2 incompatible Daemons could not be reused, use --status for details
WARNING:We recommend using a newer Android Gradle plugin to use compileSdk = 32

This Android Gradle plugin (7.0.3) was tested up to compileSdk = 31

This warning can be suppressed by adding
    android.suppressUnsupportedCompileSdk=32
to this project's gradle.properties

The build will continue, but you are strongly encouraged to update your project to
use a newer Android Gradle Plugin that has been tested with compileSdk = 32
> Task :clean
> Task :assetPacks:installTime:clean
> Task :preBuild UP-TO-DATE
> Task :preReleaseBuild UP-TO-DATE
> Task :compileReleaseRenderscript NO-SOURCE
> Task :assetPacks:installTime:generateAssetPackManifest
> Task :generateReleaseResValues
> Task :generateReleaseResources
> Task :processReleaseAssetPackManifests
> Task :linkReleaseManifestForAssetPacks
> Task :assetPackReleasePreBundleTask
> Task :createReleaseCompatibleScreenManifests
> Task :extractDeepLinksRelease
> Task :processReleaseMainManifest
> Task :processReleaseManifest
> Task :processApplicationManifestReleaseForBundle
> Task :checkReleaseDuplicateClasses
> Task :buildKotlinToolingMetadata
> Task :compileReleaseAidl NO-SOURCE
> Task :generateReleaseBuildConfig
> Task :checkReleaseAarMetadata
> Task :processReleaseManifestForPackage
> Task :javaPreCompileRelease
> Task :desugarReleaseFileDependencies
> Task :mergeReleaseShaders
> Task :compileReleaseShaders NO-SOURCE
> Task :generateReleaseAssets UP-TO-DATE
> Task :mergeReleaseAssets
> Task :processReleaseJavaRes NO-SOURCE
> Task :mergeReleaseJniLibFolders
> Task :mergeReleaseArtProfile UP-TO-DATE
> Task :mergeReleaseResources
> Task :collectReleaseDependencies
> Task :configureReleaseDependencies
> Task :parseReleaseIntegrityConfig UP-TO-DATE
> Task :writeReleaseAppMetadata
> Task :validateSigningRelease
> Task :bundleReleaseResources
> Task :mergeReleaseNativeLibs
> Task :processReleaseResources

> Task :stripReleaseDebugSymbols
Unable to strip the following libraries, packaging them as they are: libc++_shared.so, libgodot_android.so.

> Task :mergeExtDexRelease
> Task :compileReleaseKotlin
> Task :compileReleaseJavaWithJavac
> Task :dexBuilderRelease

> Task :extractReleaseNativeSymbolTables
Unable to extract native debug metadata from C:\allfiles\Godot\projects\KvsZ NOTadMob\android\build\build\intermediates\merged_native_libs\release\out\lib\arm64-v8a\libc++_shared.so because unable to locate the objcopy executable for the arm64-v8a ABI.
Unable to extract native debug metadata from C:\allfiles\Godot\projects\KvsZ NOTadMob\android\build\build\intermediates\merged_native_libs\release\out\lib\arm64-v8a\libgodot_android.so because unable to locate the objcopy executable for the arm64-v8a ABI.
Unable to extract native debug metadata from C:\allfiles\Godot\projects\KvsZ NOTadMob\android\build\build\intermediates\merged_native_libs\release\out\lib\armeabi-v7a\libc++_shared.so because unable to locate the objcopy executable for the armeabi-v7a ABI.
Unable to extract native debug metadata from C:\allfiles\Godot\projects\KvsZ NOTadMob\android\build\build\intermediates\merged_native_libs\release\out\lib\armeabi-v7a\libgodot_android.so because unable to locate the objcopy executable for the armeabi-v7a ABI.
Unable to extract native debug metadata from C:\allfiles\Godot\projects\KvsZ NOTadMob\android\build\build\intermediates\merged_native_libs\release\out\lib\x86\libc++_shared.so because unable to locate the objcopy executable for the x86 ABI.
Unable to extract native debug metadata from C:\allfiles\Godot\projects\KvsZ NOTadMob\android\build\build\intermediates\merged_native_libs\release\out\lib\x86\libgodot_android.so because unable to locate the objcopy executable for the x86 ABI.
Unable to extract native debug metadata from C:\allfiles\Godot\projects\KvsZ NOTadMob\android\build\build\intermediates\merged_native_libs\release\out\lib\x86_64\libc++_shared.so because unable to locate the objcopy executable for the x86_64 ABI.
Unable to extract native debug metadata from C:\allfiles\Godot\projects\KvsZ NOTadMob\android\build\build\intermediates\merged_native_libs\release\out\lib\x86_64\libgodot_android.so because unable to locate the objcopy executable for the x86_64 ABI.

> Task :mergeReleaseJavaResource
> Task :mergeDexRelease
> Task :buildReleasePreBundle
> Task :compileReleaseArtProfile UP-TO-DATE
> Task :packageReleaseBundle
> Task :signReleaseBundle FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':signReleaseBundle'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.FinalizeBundleTask$BundleToolRunnable
   > Failed to generate v1 signature

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 59s
44 actionable tasks: 41 executed, 3 up-to-date

Exit code: 1

### Minimal reproduction project

N/A

**Comment by GrAfDeveloper** [comment]
I finally solved this problem myself! I don't know what exactly it was related to, but I just recreated keystore.jks and this time used this command:
keytool -genkeypair -alias upload -keyalg RSA -keysize 2048 -validity 9125 -keystore keystore.jks
I used a different command before this.
It is also possible that the problem was in the name (before it was "release.jks", now it is "keystore.jks") Thank you.
